### PR TITLE
Fix workspace url links

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Fix workspace url links

### What issues does this PR fix or reference?
Fixes https://github.com/eclipse/che/issues/7833

#### Release Notes
Fix workspace url links

#### Docs PR
n/a

### CODE REVIEW

File: wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceLinksGenerator.java [hunk 1]
```diff
@@ -60,7 +60,6 @@ public WorkspaceLinksGenerator(
         LINK_REL_SELF,
         uriBuilder
             .clone()
-            .path(WorkspaceService.class)
```
Comment (sleshchenko) [inline wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceLinksGenerator.java:63]: I'm OK with it if it fixes the issue. But it is still unclear why it works fine for [User API](https://github.com/eclipse/che/blob/6e9825c699d34eef6b8bf36fc0194ff871cbee84/wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserLinksInjector.java#L49) in the same way.

**Comment by skabashnyuk** [comment]
ci-test

**Review by sleshchenko — APPROVED** [review]

**Review by garagatyi — APPROVED** [review]

File: wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceLinksGenerator.java (hunk 1 above)
Comment (skabashnyuk) [reply wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceLinksGenerator.java:63]: -   ```serviceContext.getServiceUriBuilder()``` in WorkspaceLinksGenerator
-  ```serviceContext.getBaseUriBuilder()``` in UserLinksInjector

Comment (sleshchenko) [reply wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceLinksGenerator.java:63]: I see. Thx.

**Comment by skabashnyuk** [comment]
ci-test

**Review by artaleks9 — APPROVED** [review]

**Pull request merged**